data/jibauni/jibauni 0 kanlanmo transport.txt

```diff
@@ -0,0 +1,665 @@
+				goto pinkship
+			label alphas
+			`	You begin the story, telling everything you know about how the Alphas tried to enslave human space.`
+			`	At the end, he says, "Wow. That is a long story. And I take it you are not on their side?"`
+			choice
+				`	"No, of course not."`
+					goto alphasoffer
+				`	"Yes, I have been friendly with them for years."`
+					goto friendlyalpha
+			label alphasoffer
+			`	"Good. Well, that just leaves one more matter to be wrapped up. Can you take me to <planet> in the <system> system? Kanlanmo of the Jibauni will be grateful."`
+			choice
+				`	"Of course!"`
+					accept
+				`	"Why?"`
+					goto whychoice
+				`	"Not yet, I have something I need to do first. But I'll be back shortly!"`
+					decline
+			label rude
+			`	The alien announces, though to what you cannot see, "The stranger has been ignominiously impolite." It turns to you. "What do you say, 'weirdo'? Will you apologize? Or not, and see what happens?"`
+			choice
+				`	"I apologize, and did not mean to insult you."`
+					goto rudeentry
+				`	"Why apologize to a bunch of dinky T-Rex's?"`
+					goto deathexplode
+			label deathexplode
+			`	A cluster of anti-ship weaponry is fired from all sides, immediately exploding the <ship> and everything inside it. It is sad your life had to end in such a way.`
+				die
+			label deathshot
+			`	You pull your sidearm from its holster, but before you can fire it, hidden anti-ship weaponry trains fire on your ship. Caught in crossfire, the <ship> immediately explodes, blowing you and everything in it sky high.`
+				die
+			label friendlyalpha
+			action
+				"reputation: Jibauni" = -1000
+				"reputation: Jibauni (Civilian)" = -1000
+				"reputation: Jibauni (Fueler)" = -1000
+				"reputation: Jibauni (Rebel)" = -1000
+			`	"Really? In that case, we must ask you to leave our planet immediately and never come back. Goodbye, and don't let us see your face ever again!"`
+				flee
+			label whychoice
+			action
+				set "jibauni: first contact: why"
+			`	"Because I have some business to transact, something about the Rebel Jibauni."`
+			choice
+				`	"Who are the Rebel Jibauni?"`
+					goto rebeljibauni
+				`	"Sounds good. I'll take you."`
+					accept
+				`	"No. If it has to do with another faction, I can't."`
+					decline
+			label rebeljibauni
+			`	"They are, as the name might suggest, Jibauni who have split away from the real government. They are not very polite, and you definitely do not want to be on their side."`
+			choice
+				`	"Okay. I'll take you."`
+					accept
+				`	"That's what you think. Well, I'm joining the Rebels, whatever you may say."`
+					goto joinedrebels
+				`	"No, I won't take you. Sorry."`
+					decline
+			label joinedrebels
+			action
```

Comment:
Because they'll notice you coming out of Jibauni space with hostile ships trailing you.

If they haven't received an Alpha message, they'll assume you were rude to the Jibauni... and anybody who does that I surely a friend of the Rebels.